IDE/WIN/user_settings_dtls.h

```diff
@@ -0,0 +1,98 @@
+#ifndef _WIN_USER_SETTINGS_H_
```

Comment:
You need to add a section in the IDE/WIN/README.md describing how to use this... Example: Rename to user_settings.h.